sktime/forecasting/naive.py

```diff
@@ -20,26 +20,36 @@ class NaiveForecaster(OptionalForecastingHorizonMixin, BaseLastWindowForecaster)
 
     Parameters
     ----------
-    strategy : str{"last", "mean", "seasonal_last"}, optional (default="last")
+    seasonal : True or False or None, optional (default=None)
+        Seasonal used to determine forecasts:
+        *True : forecast the seasonal type of given strategy
+        *False : forecast the given strategy itself
+
+    strategy : str{"last", "mean"}, optional (default="last")
         Strategy used to make forecasts:
 
         * "last": forecast the last value in the training series
         * "mean": forecast the mean of (a given window) of the training series
-        * "seasonal_last": forecasts the last value of the same season in the training series
-
+        *  seasonal is True and "last": forecasts the last value of the same season in the training series
+        * seasonal is True and "mean": forecast the mean (of a given window) of the same season in the training series
```

Comment:
Yes, so the meaning of window changes slightly depending on whether seasonal is true or false:
* if seasonal=True, it refers to the number of previous seasons that we compute the mean over
* if seasonal=False, it refers to the number of previous time points we compute the mean over